src/main/setup.hpp

```diff
@@ -4,8 +4,15 @@
 #include "SDL_video.h"
 const static int SDL_FLAGS = SDL_SWSURFACE | SDL_DOUBLEBUF;
 #endif
+#ifndef __CELLOS_LV2__
 const static char* FILENAME_CONFIG = "./config.xml";
 const static char* FILENAME_SCORES = "./hiscores";
 const static char* FILENAME_TTRIAL = "./hiscores_timetrial";
 const static char* FILENAME_CONT   = "./hiscores_continuous";
+#else
```

Comment:
I will likely have to make my own commit to do this properly. Paths like this should not be hardcoded and they should properly query the SYSTEM DIR path.

Reply:
So, do you want i remove it from PR?